src/cmd-run

```diff
@@ -173,7 +173,12 @@ EOF
         exit 1
     fi
 fi
-set -- -fw_cfg name=opt/com.coreos/config,file="${IGNITION_CONFIG_FILE}" "$@"
+
+case $arch in
+x86_64|aarch64) set -- -fw_cfg name=opt/com.coreos/config,file="${IGNITION_CONFIG_FILE}" "$@" ;;
+# no qemu_fw_cfg on these architectures just run the image without the configuration
+ppc64le|s390x) set -- "$@" ;;
```

Comment:
I like the explicit `case` to make it clear it's not supported on those platforms. Though we can just do `;;` here, right?

Reply:
Maybe some message instead in case someone is trying to test on `ppc64le` and wondering why the ignition file is not working?